packages/story-editor/src/components/header/buttons/publish.js

```diff
@@ -109,44 +99,36 @@ function PublishButton({ forceIsSaving }) {
   }, [refreshPostEditURL, saveStory, hasFutureDate, title, canPublish]);
 
   const handlePublish = useCallback(() => {
-    if (
-      canPublish &&
-      (shouldReviewDialogBeSeen || isUpdatedPublishModalEnabled)
-    ) {
+    updateToAllowPriorityIssues();
+    if (shouldReviewDialogBeSeen || isUpdatedPublishModalEnabled) {
       setShowDialog(true);
       return;
     }
 
     publish();
   }, [
+    updateToAllowPriorityIssues,
     shouldReviewDialogBeSeen,
     isUpdatedPublishModalEnabled,
-    canPublish,
     publish,
   ]);
 
   const closeDialog = useCallback(() => setShowDialog(false), []);
 
-  const text =
-    hasFutureDate && status !== 'private'
-      ? __('Schedule', 'web-stories')
-      : __('Publish', 'web-stories');
-
   return (
     <>
       <ButtonWithChecklistWarning
         onClick={handlePublish}
-        disabled={isSaving || forceIsSaving || isUploading}
```

Comment:
what was the difference between `forceIsSaving` and `isSaving`?

Reply:
great question, so `forceIsSaving is for the metaboxes specific to wp-story-editor, it gets set in `wp-story-editor/src/components/header/buttons`.

Reply:
So this button should only be disabled when metaboxes are updating?

Reply:
or when media/story is updating.

Reply:
Oh I think I see where I'm confused now:

`forceIsSaving` is for metaboxes, but for media/story updates that's tracked by `isSaving` and `isUploading`, right? Those were removed here so wondering if those need to be added back in.

Reply:
I think that `isSaving` and `isUploaded` are now defined in the  `ButtonWithChecklistWarning` component

Reply:
Ah, yup! 😮‍💨 thank you!